astro/src/content/quickstarts/_5-minute-application-setup.mdx

```diff
@@ -1,15 +1,20 @@
 import InlineField from 'src/components/InlineField.astro';
 import InlineUIElement from 'src/components/InlineUIElement.astro';
 
-Once you arrive in the FusionAuth admin UI, the first thing you need to do is create an Application. An Application is something that a user can log into. This is the application you are building or that you are migrating to use FusionAuth. Click the <InlineUIElement>Applications</InlineUIElement> menu option on the left side of the page.
+Once you arrive in the FusionAuth admin UI, the first thing you need to do is create an Application. 
+
+If you see a Setup wizard in the main content window, you can click "Skip it" as you'll be creating things outside of this wizard.
```

Comment:
```suggestion
If you see a First Time Setup wizard in the main content window, you can click "Skip it" as you'll be creating things outside of this wizard.
```

setup wizard is where you set up the first admin user and consent to the license.
first time setup is what you see when you first get into the admin ui.